sandbox/registration-processor-mz.properties

```diff
@@ -815,7 +794,7 @@ mosip.regproc.biometric.authentication.server.port=8020
 mosip.regproc.biometric.authentication.eventbus.port=5777
 mosip.regproc.biometric.authentication.server.servlet.path=/registrationprocessor/v1/bioauth
 
-#-----------------------------------------reprocessor-stage----------------------------------------#
+#-----------------------------------------reprocessor----------------------------------------#
```

Comment:
We can also move this full set of reprocessor properties to be under the WORKFLOW ENGINE PROPERTIES sesction